npm/src/debug-service/debug-service.ts

```diff
@@ -164,42 +151,23 @@ export class QSharpDebugService implements IDebugService {
     });
     const positionMap = mapUtf8UnitsToUtf16Units(positions, this.code[path]);
     const breakpoint_spans: IBreakpointSpan[] = breakpoint_list.spans.map(
-      (span: IBreakpointSpan) => {
-        const result = {} as IBreakpointSpan;
-        result.id = span.id;
-        result.lo = positionMap[span.lo];
-        result.hi = positionMap[span.hi];
-        return result;
-      }
+      (span) => ({
```

Comment:
Do you need the parens around the braces?

Reply:
Yes! This is kind of an annoying JavaScript syntax ambiguity. `() => {}` and `() => ({})` mean different things - in the first one, `{}` is parsed as a block, so the lambda returns `void`. In the second one `{}` is parsed as an object declaration, so the lambda returns an `object`.

Worse, `() => { foo: 3 }` is valid code, but instead of returning an object like you would expect, it's actually declaring a code block with the labeled statement `foo` and a statement with just `3` in it, which is a valid expression therefore a valid statement. Then the block returns `void`. JavaScript will happily execute this, but at least TypeScript will complain that the label is unused. It's very common to write code like this and then stare at it wondering what TypeScript is complaining about :D